README.md

```diff
@@ -19,7 +19,7 @@ The main objective of this week is to develop the MVP feature set listed below u
 * [ ] Fill in the `backlog` list with all the extra features listed below
```

Comment:
This file is probably better left blank, or if you have specific instructions to leave behind for yourself, this would be a good place to put them. I usually fill my Readme with design specs, and software architecture decisions and notes for later implementation of additional functionality.